pyanaconda/payload/rpmostreepayload.py

```diff
@@ -206,7 +205,7 @@ def install(self):
         if hasattr(ostreesetup, 'nogpg') and ostreesetup.nogpg:
             self._remoteOptions['gpg-verify'] = Variant('b', False)
 
-        if flags.noverifyssl:
+        if not self.is_ssl_verification_enabled:
```

Comment:
The kickstart command `ostreesetup` is not in `data.method` and doesn't have the `noverifyssl` attribute anyway.

Reply:
That shouldn't be a problem. The method is skipped in case it is not present.